tools/xref/StmtPrinter.cpp

```diff
@@ -751,13 +813,25 @@ void StmtPrinter::VisitOMPParallelMasterDirective(
   PrintOMPExecutableDirective(Node);
 }
 
+void StmtPrinter::VisitOMPParallelMaskedDirective(
```

Comment:
Do we need the OpenMP pragma printers for anything?

Reply:
Just for making Clang happy and making tracking changes across versions easier